articles/TOC.yml

```diff
@@ -44,6 +44,8 @@
     href: intro-to-katas.md
   - name: Q# code samples
     href: https://docs.microsoft.com/en-us/samples/browse/?languages=qsharp
+- name: The Q# User Guide
```

Comment:
Ah yes, sorry I didn't mention that. I decided to just leave all the other sections in place while building the User Guide, so that it might be be easier to make/decide all the proper re-directs, etc. I just added a note to the PR description. 

Basically, the Q# Programming Language and the Quantum Dev. Techniques sections will be entirely deleted, as they've been fully brought into the new pages. Planning to bring the target machines section into the user guide too (although it just stands on its own anyway), and therefore the whole "in-depth" tab will be deleted. The "libraries" section will remain on its own I believe, but a page will be added to the User Guide which details and links out to each of them.

Reply:
Makes sense. I think that's another good reason to develop this targeting a feature branch while that reorganization is in progress.

Reply:
Can someone approve my  joining the network so I don't gotta bother you guys here?